src/pages/index.js

```diff
@@ -1,11 +1,7 @@
 import React from 'react';
```

Comment:
Is this deployed somewhere that I can take a look?

Reply:
I deployed it to your local machine when you were in the shower.